### PR TITLE
FIX - delimitation element handling.

### PR DESCRIPTION
**Pull request by hamish-mcc**
Standard reference https://dicom.nema.org/medical/dicom/current/output/chtml/part05/sect_7.5.html

### CODE REVIEW

File: src/ValueRepresentation.js [hunk 1]
```diff
@@ -421,13 +420,9 @@ class BinaryRepresentation extends ValueRepresentation {
                         frames.push(fragmentStream.buffer);
                     }
                 }
-
-                // Read SequenceDelimitationItem Tag
+                // Read delimitation item length (expect 00 00 00 00)
+                // https://dicom.nema.org/medical/dicom/current/output/chtml/part05/sect_7.5.html
```
Comment (richard-viney) [inline src/ValueRepresentation.js:424]: I think it'd be good to check that the 4 bytes we read are in fact zero?

File: src/ValueRepresentation.js [hunk 2]
```diff
@@ -363,7 +363,6 @@ class BinaryRepresentation extends ValueRepresentation {
                 // redundant approach.
                 if (offsets.length > 0) {
                     offsets.push(stream.size);
-
```
Comment (richard-viney) [inline src/ValueRepresentation.js:366]: I try to avoid noise like this in PRs to public projects

File: src/ValueRepresentation.js [hunk 3]
```diff
@@ -421,13 +420,9 @@ class BinaryRepresentation extends ValueRepresentation {
                         frames.push(fragmentStream.buffer);
                     }
                 }
-
-                // Read SequenceDelimitationItem Tag
+                // Read delimitation item length (expect 00 00 00 00)
```
Comment (richard-viney) [inline src/ValueRepresentation.js:423]: In what sense is this a "length"?

**Comment by hamish-mcc** [comment]
More investigation required. Will submit to main repo as an issue.

**Pull request closed without merging**